core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/LambdaVariables/VariableInputs/ResourceVariableInput.js

```diff
@@ -240,6 +260,11 @@ export default function ResourceVariableInput({
               }
             />
           </FormItem>
+          {takeAll && (
+            <MessageStrip className="fd-margin-top--sm" type="information">
+              {t('functions.variable.form.name-will-become-prefix')}
```

Comment:
Maybe we can add some information that if ALL KEYS is selected multiple variables will be created (one for every existing secret key)?

Reply:
Not a bad idea! I'll add it to this string.